test-script/README.md

```diff
@@ -0,0 +1,22 @@
+## Run test configs in parallel on multiple workers
+
+### How to run
+1. Finish the learning mode and generate all the test configs on a master node(could be any worker node)
```

Comment:
Just a reminder, when experimenting we should always run the learning mode on the vms instead of PT1/2/3 as we want to ensure the env for sieve learn/test is the same.

Reply:
Exactly, by master node I mean any of the vms.